frontend/packages/shipwright-plugin/integration-tests/support/step-definitions/builds/shipwright-build-detail-page.ts

```diff
@@ -20,6 +22,10 @@ When('user has created shipwright builds', () => {
   });
 });
 
+Given('user is at Add page', () => {
```

Comment:
Move it to [common.ts](https://github.com/openshift/console/pull/11889/files#diff-cc9c6e7a16ce36aba7b3961a5cc2e787acc89c404c9a844d310bf39e79ef764b)